src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/__tests__/PPAnalyticsSubmitterTests.swift

```diff
@@ -0,0 +1,320 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class PPAnalyticsSubmitterTests: XCTestCase {
+
+	// MARK: - Success
+
+	func testGIVEN_SubmissionIsTriggered_WHEN_EverythingIsGiven_THEN_Success() {
+		// GIVEN
+		let store = MockTestStore()
+		let client = ClientMock()
+		var config = SAP_Internal_V2_ApplicationConfigurationIOS()
+		config.privacyPreservingAnalyticsParameters.common.probabilityToSubmit = 3
```

Comment:
Shouldn't be `probabilityToSubmit ` not in 0...1?

Reply:
The probability flag has to be "disabled" here to test other things. If the value is 3, the check will always success